content/05.conclusion.md

```diff
@@ -1,19 +1,28 @@
-## Conclusions
+## Discussion and Conclusions
+
+BioRxiv is a popular repository that contains life science preprints; however, the language contained within these preprints remains unknown.
+Throughout this work we sought to analyze the language contained within these preprints and understand how it changes in response to the peer review process.
+Through our global corpora analysis, we found that writing within bioRxiv is consistent with the biomedical literature contained in the PMCOA repository, suggesting that bioRxiv is linguistically similar to PMCOA.
+Token-level analyses between bioRxiv and PMCOA suggested that major differences are driven by research fields where more patient related research is prevalent in PMCOA than bioRxiv.
+This observation is expected as patient preprints are funneled from bioRxiv into the medRxiv repository [@url:https://www.medrxiv.org/].
+Through our document embedding approach., we observed that life science preprints were separated on an informatics / molecular axis, suggesting that majority of life science research involves some form of a cellular biology or an informatics type of approach.

```

Comment:
```suggestion
Examining document embeddings revealed that the largest source of variability for life science preprints was an informatics / molecular axis.
```
I'm not sure the second part of that statement is really true because you show above that bioRxiv and PMC don't really have quite the same content.